### PR TITLE
Remove deprecated Gem::Specification#has_rdoc

### PR DESCRIPTION
**Pull request by jcoyne**
```
NOTE: Gem::Specification#has_rdoc= is deprecated with no replacement. It will be removed on or after 2018-12-01.
Gem::Specification#has_rdoc= called from /home/travis/build/sul-dlss/argo/vendor/bundle/ruby/2.5.0/bundler/gems/rdf-rdfa-240ca8bc11c8/rdf-rdfa.gemspec:21
```

### CODE REVIEW

**Comment by jcoyne** [comment]
@gkellogg I'm not sure how to fix the rbx builds. Can I move them to allowed failures?

**Comment by gkellogg** [comment]
Yes, rbx is ignored everywhere else, as is jRuby. They both have seemingly unresolvable issues across the board.

File: .travis.yml [hunk 1]
```diff
@@ -6,11 +6,14 @@ env:
 rvm:
   - 2.0
   - 2.1
-  - 2.2.1
+  - 2.2.7
```
Comment (gkellogg) [inline .travis.yml]: We should add 2.2.2, since I've seen cases where stuff worked in 2.2.7, but not in 2.2.2.

Comment (jcoyne) [reply .travis.yml]: Okay, it's been re-added.

**Comment by gkellogg** [comment]
Thanks @jcoyne. I'll wait for #26 to be rebased on this to merge and release an update.

**Pull request merged**